docs/mindsdb-docs/docs/sql/tutorials/customer-churn.md

```diff
@@ -1,127 +1,196 @@
++----------+
+| status   |
++----------+
+| complete |
++----------+
 ```
 
-![select](/assets/sql/tutorials/customer_churn/select.png)
+Now, if the status of our predictor says `complete`, we can start making predictions!
+
+## Making Predictions
+
+You can make predictions by querying the predictor as if it were a table. The [`SELECT`](/sql/api/select/) syntax lets you make predictions for the label based on the chosen features.
+
+```sql
+SELECT Churn, Churn_confidence, Churn_explain
+FROM mindsdb.customer_churn_predictor
+WHERE SeniorCitizen=0 AND Partner='Yes' AND Dependents='No' AND tenure=1 AND PhoneService='No' AND MultipleLines='No phone service' AND InternetService='DSL';
+```
 
-The next steps would be to query the model and predict the customer churn. Let’s be creative and imagine a customer. Customer will use only DSL service, no phone service and multiple lines, she was with the company for 1 month and has a partner. Add all of this information to the WHERE clause.
+On execution, you get the following output:
 
 ```sql
-SELECT Churn, Churn_confidence, Churn_explain FROM mindsdb.customer_churn WHERE SeniorCitizen=0 AND Partner='Yes' 
-AND Dependents='No' AND tenure=1 AND PhoneService='No' AND MultipleLines='No phone service' AND InternetService='DSL';
++-------+---------------------+-------------------------------------------------------------------------------------------------+
+| Churn | Churn_confidence    | Churn_explain                                                                                   |
++-------+---------------------+-------------------------------------------------------------------------------------------------+
+| Yes   | 0.7865168539325843  | {"predicted_value": "Yes", "confidence": 0.7865168539325843, "anomaly": null, "truth": null}    |
++-------+---------------------+-------------------------------------------------------------------------------------------------+
 ```
 
-![customer churn](/assets/sql/tutorials/customer_churn/customer_churn.png)
+Let's try another prediction.
+
+An important thing to check is the `important_missing_information` value, where MindsDB points to the important missing information that should be included to give a more accurate prediction. In this case, the `Contract`, `MonthlyCharges`, `TotalCharges`, and `OnlineBackup` columns are the important missing information. Let’s include those values in the `WHERE` clause, and run a new query.
+
+```sql
+SELECT Churn, Churn_confidence, Churn_explain
+FROM mindsdb.customer_churn_predictor
+WHERE SeniorCitizen=0 AND Partner='Yes' AND Dependents='No' AND tenure=1 AND PhoneService='No' AND MultipleLines='No phone service' AND InternetService='DSL' AND OnlineSecurity='No' AND OnlineBackup='Yes' AND DeviceProtection='No' AND TechSupport='No' AND StreamingTV='No' AND StreamingMovies='No' AND Contract='Month-to-month' AND PaperlessBilling='Yes' AND PaymentMethod='Electronic check' AND MonthlyCharges=29.85 AND TotalCharges=29.85;
```

Comment:
Break into new lines each AND for readability.

Reply:
Done.